utility.py

```diff
@@ -264,3 +267,46 @@ def tag_user(user, tag, channel):
                 queue_message_to_one(messages.tag_message(get_display_name(user_id), tag), channel)
             else:
                 queue_message_to_one(messages.already_tag_message(get_display_name(user_id), tag), channel)
+
+### Banphrase API ###
+
+def check_banphrase(message, channel_name):
+    headers = { 'User-Agent': 'huwobot (https://huwobot.com/)' }
+    params = (('message', message),)
+
+    banphrase_api =  db(opt.CHANNELS).find_one({'name': channel_name})['banphrase_api']
+
+    if len(banphrase_api) == 0:
+        return False
+
+    try:
+        time.sleep(random.uniform(0.1, 1))
```

Comment:
why this sleep?

Reply:
https://github.com/huwodro/dungeonbot/blob/5443cd1ba7c5c01a1351e3140e1875f5c590452d/bot.py#L117

This call goes through every leveled up user, the sleep is to make sure the API doesn't die.